### PR TITLE
PR-15192: Added applicationIdentifier field

### PR DESCRIPTION
**Pull request by robertogrings**
#### Description
Added applicationIdentifier field on VendorFieldValidationRequest for the VV API

#### Manual merge checklist:
- [ ] Code Review completed
- [ ] Code coverage
- [ ] QA Validation completed
  - Testrail TestCase/Plan link:
- [ ] Approved by a PI tech lead
- [ ] Github checks all green

### CODE REVIEW

**Comment by naponce** [comment]
LGTM!

File: src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java [hunk 1]
```diff
@@ -30,28 +28,29 @@
...
-
-	@InitBinder
-	public void initBinder(final WebDataBinder webdataBinder) {
-		webdataBinder.registerCustomEditor(FlowType.class, new FlowTypeConverter());
-		webdataBinder.registerCustomEditor(OperationType.class, new OperationTypeConverter());
-	}
+    @Autowired
+    private final VendorFieldValidationHandler vendorFieldValidationHandler;
+
+    @RequestMapping(
```
Comment (gastontulipani) [inline src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java]: This indentation is from spaces. Please revert all indentation changes and fix your intellij configuration

Comment (robertogrings) [reply src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java]: Done

File: src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java [hunk 2]
```diff
@@ -1,27 +1,25 @@
 package com.appdirect.sdk.vendorFields.controller;
 
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
```
Comment (gastontulipani) [inline src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java]: Please review import order

Comment (robertogrings) [reply src/main/java/com/appdirect/sdk/vendorFields/controller/VendorFieldValidationController.java]: Done

**Pull request merged**